apps/kserve/data.yaml

```diff
@@ -0,0 +1,79 @@
+tags:
+  - AI/Machine Learning
+  - Serverless
+title: KServe
+summary: "Serverless platform for deploying, monitoring, and managing ML models on Kubernetes."
+logo: ./assets/kserve.png
```

Comment:
We can use just svg logo version as the png one is the same anyway.

Reply:
Done.